app/code/Magento/Checkout/view/frontend/web/js/model/checkout-data-resolver.js

```diff
@@ -257,6 +263,16 @@ define([
 
             shippingAddress = quote.shippingAddress();
 
+            customerAddressList = customerAddress.getAddressItems();
+            $.each(customerAddressList, function (key, item) {
```

Comment:
Why not use an underscore foreach? This way we don't need to require the extra jQuery only for a foreach loop when underscore is already required.